pkgs/http/CHANGELOG.md

```diff
@@ -1,234 +1,238 @@
+## 1.1.3
+
+- Document HTTP client response header handling
```

Comment:
It looks like you replaced "*" with "-" - it is "*" at HEAD: https://github.com/dart-lang/http/blob/master/pkgs/http/CHANGELOG.md?plain=1

Could you change the version to:

## 1.1.3-wip

And make the same change in pubspec.yaml